public/data/dapoadedire.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Dapo Adedire",
+  "type": "personal",
+  "bio": "Backend Developer and Open Source Enthusiast!",
+  "avatar": "https://github.com/dapoadedire.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/dapoadedire",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/dapo_adedire",
+      "icon": "twitter"
+    }
+  ],
```

This will fix the CI (continuous integration) errors. :)
```suggestion
  ]
```